git-mit/man-template/git-mit.1.md

```diff
@@ -0,0 +1,98 @@
+% GIT-MIT(1) Version 3.12.0 | The git-mit suite of git hooks
+
+NAME
+====
+
+**git-mit** — Set author and Co-authored trailer.
+
+SYNOPSIS
+========
+
+| git-mit [-h|--help|-v|--version]
+| git-mit [OPTIONS] <initials>...
+
+DESCRIPTION
+===========
+
+Set author and Co-authored trailer.
+
+Arguments
+---------
+
+\<initials\>...
+
+:   Initials of the author to put in the commit
+
+Options
+-------
+
+-h, --help
+
+:   Prints help information
+
+-V, --version
+
+:   Prints version information
+
+-e, --exec \<command\>
+
+:  Execute a command to generate the author configuration, stdout will be captured and used instead of the file, if both this and the file is present, this takes precedence 
+
+-c, --config \<file\>
+
+:  Path to a file where author initials, emails and names can be found 
+
+-t, --timeout \<timeout\>
+
+:  Number of minutes to expire the configuration in 
+
+FILES
+=====
+
+*~/.config/git-mit/mit.yml*
```

Comment:
I guess this section is just saved as a text file or in a string somewhere. We can't work it out from the app